web/profiles/custom/yalesites_profile/config/sync/field.field.media.video.field_media_oembed_video.yml

```diff
@@ -0,0 +1,19 @@
+uuid: c02baedc-ded8-403e-bfa9-1bc3273f39e0
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.media.field_media_oembed_video
```

Comment:
Please change the machine name of the field `field_media_url`. Information about media fields is in the data model spreadsheet on a separate page so it may have been easy to miss. You may need to delete and re-add this field to make the channge.